C/plugins/storage/sqlite/common/connection.cpp

```diff
@@ -1189,9 +1189,11 @@ vector<string>  asset_codes;
  */
 int Connection::insert(const string& schema, const string& table, const string& data)
 {
-SQLBuffer	sql;
+string sqlCommand = {};
```

Comment:
Do not change away from using the SQLBuffer class, this was done explicitly for this purpose to improve performance by reducing string copies.

Reply:
fixed